frontend/src/state/GraphState.jsx

```diff
@@ -490,70 +508,84 @@ export class GraphContainer {
       },
     }
     for (let type in rows) {
-      let row = {...get(this, ['statsOptions', type], {}), ...rows[type]};  // don't lose stuff previously set
+      let row = {...get(this, ['graphDisplayConfig', type], {}), ...rows[type]};  // don't lose stuff previously set
       if (typeof(row.value) === 'undefined') {  // don't show rows that don't represent any concepts
         delete rows[type];
         continue;
       }
       row.type = type;
-      if (isEmpty(this.statsOptions) && typeof(row.specialTreatmentDefault) !== 'undefined') {
+      if (isEmpty(this.graphDisplayConfig) && typeof(row.specialTreatmentDefault) !== 'undefined') {
         // set specialTreatment to default only the first time through
         row.specialTreatment = row.specialTreatmentDefault;
       }
-      // TODO: figure out if row.specialTreatment and
-      //       this.options.specialConceptTreatment are redundant
-      //       OR, maybe they are but saving it in this.options is good
-      //       for saving the options
-      // TODO: implement saving and reloading the options
-      if (type in this.gd.specialConcepts) {
+      // if (type in this.gd.specialConcepts && graphOptions.specialConceptTreatment[type] === undefined) {
+        // don't mutate graphOptions
+        // graphOptions = {...graphOptions};
         // don't bother setting the option if there are no concepts/items of this type
-        this.options.specialConceptTreatment[type] = row.specialTreatment;
-      }
+        // this.options.specialConceptTreatment[type] = row.specialTreatment;
+        // graphOptions.specialConceptTreatment[type] = row.specialTreatmentDefault;
+      // }
       rows[type] = row;
     }
-    this.statsOptions = rows;
+    this.graphDisplayConfig = rows;
+    this.graphDisplayConfigList = sortBy(rows, d => d.displayOrder);
```

Comment:
## Row display order
@Sigfried Saw in [this commit](https://github.com/jhu-bids/TermHub/commit/050067ce41f9bb06fae763c03a3f3882669058a8). Is this change about sorting by column?